contrib/for-tests/netexec/pod.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: netexec
+  labels:
+    app: netexec
+spec:
+  containers:
+  - name: netexec
+    image: stevemilnerrh/netexec:1.2
```

Comment:
I actually think, having written a few E2E tests that had dependencies on image changes, that for now, it's good to keep the test image in there.  Add a TODO yes, but don't bother changing it until you have some OK from upstream on the rest of the test.  In the TODO explain that you'll change it before commit upstream.